.eslintrc

```diff
@@ -55,10 +55,32 @@
   },
   "overrides": [
     {
-      "files": ["*.ts", "*.tsx"],
+      "files": ["src/**/*.{ts,tsx}"],
+      "plugins": [
+        "@typescript-eslint",
+      ],
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "project": ["./tsconfig.json"],
+      },
       "rules": {
-        "no-undef": "off"
-      }
+        "no-undef": "off",
+        "@typescript-eslint/no-unused-vars": [
+          "warn",
+          {
+            "vars": "all",
+            "args": "all",
+            "argsIgnorePattern": "_|^_|^props",
+            "destructuredArrayIgnorePattern": "_|^_",
+            "ignoreRestSiblings": true
+          }
+        ]
```

Comment:
I think we should only ignore unused variables that start with `_`. Per the eslint docs, the recommended syntax for this is:

```
    "@typescript-eslint/no-unused-vars": [
      "error",
      {
        "args": "all",
        "argsIgnorePattern": "^_",
        "caughtErrors": "all",
        "caughtErrorsIgnorePattern": "^_",
        "destructuredArrayIgnorePattern": "^_",
        "varsIgnorePattern": "^_",
        "ignoreRestSiblings": true
      }
    ]
```

Reply:
Sounds good to me. It seems like `caughtErrors` already defaults to `all`, so I won't explicitly add that.